content/en/smart-contracts/filecoin-evm-runtime/precompiles/index.md

```diff
@@ -92,7 +92,7 @@ Calls the specified actor using the native FVM calling convention by its _Fileco
   - `exit_code` is one of:
     - `= 0` to indicate the call exited successfully.
     - `> 0` to indicate that the target actor _referted_ with the specified `exit_code`.
```

Comment:
For the life of me, I can't figure out a) what this word was supposed to be if it is wrong or b) if it is some special word that I've never heard before.

@johnnymatthews 😖

Reply:
```suggestion
    - `> 0` to indicate that the target actor _reverted_ with the specified `exit_code`.
```